source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -10,28 +10,51 @@ Bulk Operations
    :depth: 2
    :class: singlecol
 
+.. meta::
+   :keywords: insert, update, replace, code example, efficiency
+
 Overview
 --------
 
 In this guide, you can learn how to use bulk operations in the
 MongoDB Java Driver. 
 
-To perform a create, replace, update, or delete operation,
-use its corresponding method. For example, to insert one document,
-update multiple documents, and delete one document in your collection,
-use the ``insertOne()``, ``updateMany()`` and ``deleteOne()`` methods.
+To perform a single create, replace, update, or delete operation, you can use
+the corresponding method. For example, to insert one document and replace one 
+document, you can use the ``insertOne()`` and ``replaceOne()`` methods. In this
+case, the ``MongoClient`` makes a call to the database for each operation. 
+
+Generally, you can reduce the number of calls to the database by using bulk write 
+operations. You can perform bulk write operations at the following levels:
+
+- :ref:`Collection Level <java-sync-coll-bulk-write>`: You can use the 
+  ``MongoCollection.bulkWrite()`` method to perform bulk write operations on a 
+  single collection. This method groups write operations together by the kind
+  of operation. For example, ``MongoCollection.bulkWrite()`` puts multiple insert
+  operations, including ``insertOne()`` and ``insertMany()``, in the same call, 
+  but will make two calls to the database for an insert operation and a replace 
+  operation.
```

Comment:
The `insertOne()` and `insertMany()` methods here may refer either to the `MongoCollection` methods, or to the `ClientNamespacedWriteModel` methods, but there are no such methods in the collection bulk write API. We should refer to the proper program elements (note that there is no `InsertManyModel`):

```suggestion
  of operation. For example, ``MongoCollection.bulkWrite()`` puts multiple insert
  operations, including ``UpdateOneModel`` and ``UpdateManyModel``, in the same call, 
  but will make two calls to the database for an `UpdateOneModel` a `ReplaceOneModel`.
```